src/index.js

```diff
@@ -0,0 +1,108 @@
+import './main.css';
+import clearListContainer from './utils/clear-container.js';
+import checkAndUncheckTodo from './utils/check-uncheck-todo.js';
+import clearAllCompletedTodos from './utils/clear-all-completed-todos.js';
+import editTodo from './utils/edit.js';
+
+// Get HTML elements
+const listContainer = document.querySelector('[data-lists]');
+const newTodoForm = document.querySelector('[data-new-todo-form]');
+const newTodoInput = document.querySelector('[data-new-todo-input]');
+
+// Get clear all completed button
+const clearButton = document.querySelector('[data-clear-all]');
+
+// Local storage data
+const localStorageTodos = 'todo.lists';
+const todos = JSON.parse(localStorage.getItem(localStorageTodos)) || [];
+
+const createList = (name) => ({
+  id: Date.now().toString(),
+  description: name,
+  completed: false,
+  index: todos.length + 1,
+});
```

Comment:
Ok, sure.
I actually was using ID since the start of the project when looping and parsing data to the HTML.
I'll see around that and adjust the places I made use of the ID and use the index instead. Thanks for your review brother.

Reply:
I have adjusted the code and removed the id property. Thanks for picking that up and for the complement. 🤝